src/components/notebook-utils.js

```diff
@@ -69,8 +69,8 @@ export const analysisNameInput = ({ inputProps, ...props }) => h(ValidatedInput,
 })
 
 export const tools = {
-  RStudio: { label: 'RStudio', ext: 'Rmd', imageIds: ['terra-jupyter-bioconductor', 'RStudio', 'terra-jupyter-bioconductor_legacy'], defaultImageId: 'terra-jupyter-bioconductor' },
-  Jupyter: { label: 'Jupyter', ext: 'ipynb', imageIds: ['terra-jupyter-hail', 'terra-jupyter-python', 'terra-jupyter-gatk', 'Pegasus', 'terra-jupyter-gatk_legacy'], defaultImageId: 'terra-jupyter-gatk' },
+  RStudio: { label: 'RStudio', ext: 'Rmd', imageIds: ['RStudio'], defaultImageId: 'RStudio' },
```

Comment:
this was a bug, in that the default RStudio image was not actually an RStudio image